scripts/bc-wp-init.sh

```diff
@@ -12,6 +14,31 @@ else
     sed -i -e "/enable_php56:/s/.*/enable_php56: false/" site.yml
 fi
 
+echo "Evaluating ip address for new vagrant box..."
+nextip(){
+    IP=$1
+    IP_HEX=$(printf '%.2X%.2X%.2X%.2X\n' `echo $IP | sed -e 's/\./ /g'`)
+    NEXT_IP_HEX=$(printf %.8X `echo $(( 0x$IP_HEX + 1 ))`)
+    NEXT_IP=$(printf '%d.%d.%d.%d\n' `echo $NEXT_IP_HEX | sed -r 's/(..)/0x\1 /g'`)
```

Comment:
This line is not working on Mac:
```$ NEXT_IP=$(printf '%d.%d.%d.%d\n' `echo $NEXT_IP_HEX | sed -r 's/(..)/0x\1 /g'`)
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
```

Reply:
Fixed. Replace `sed -r` to mac-compatible command.